src/ngx_http_lua_exitworkerby.c

```diff
@@ -0,0 +1,60 @@
+
+/*
+ * Copyright (C) Xianliang Li
```

Comment:
For this patch to merge, you have to grant your copyright to the authors of this module. So please remove this copyright notice comment.